packages/labs/src/components/omnibox/omnibox.tsx

```diff
@@ -179,20 +184,32 @@ export class Omnibox<T> extends React.Component<IOmniboxProps<T>, IOmniboxState<
     }
 
     private maybeRenderMenu(listProps: IQueryListRendererProps<T>) {
-        if (this.state.query.length > 0) {
+        const { initialContent } = this.props;
+        let menuChildren: any;
+
+        if (!this.isQueryEmpty()) {
+            menuChildren = this.renderItems(listProps);
+        } else if (initialContent != null) {
+            menuChildren = initialContent;
+        }
+
+        if (menuChildren != null) {
             return (
                 <Menu ulRef={listProps.itemsParentRef}>
-                    {this.renderItems(listProps)}
+                    {menuChildren}
                 </Menu>
             );
         }
+
         return undefined;
     }
 
+    private isQueryEmpty = () => this.state.query.length === 0;
```

Comment:
I prefer explicit `return` if you're returning something. and explicit returns inline are kinda weird (TSLint may even complain about that), so here's how I would write this:

```tsx
private isQueryEmpty = () => {
    return this.state.query.length === 0;
}
```

Reply:
generally 👎 ; I think the terse style is nice